build/web_compose/compose.sh

```diff
@@ -19,21 +19,47 @@ export TOKENIZERS_PARALLELISM=false
 # https://github.com/UKPLab/sentence-transformers/issues/1318#issuecomment-1084731111
 export OMP_NUM_THREADS=1
 
-if [[ $USE_SQLITE == "true" ]]; then
-    export SQLITE_PATH=/sqlite/datacommons.db
+# If OUTPUT_DIR is not specified and the deprecated GCS_DATA_PATH is, use that as OUTPUT_DIR.
+if [[ $OUTPUT_DIR == "" && $GCS_DATA_PATH != "" ]]; then
+    echo "GCS Data Path: $GCS_DATA_PATH"
+    echo "GCS_DATA_PATH is deprecated and will be removed in the future. Use OUTPUT_DIR instead."
+    export OUTPUT_DIR=$GCS_DATA_PATH
 fi
 
-nginx -c /workspace/nginx.conf
+# Set OUTPUT_DIR to same as INPUT_DIR if not specified.
+if [[ $OUTPUT_DIR == "" ]]; then
+    if [[ $INPUT_DIR == "" ]]; then
```

Comment:
Should we do this check even if OUTPUT_DIR is set?

Reply:
Probably not. This docker ("services docker") is about serving data from the output directory so if it is set, that's the one we want to use.